addon-test-support/@percy/ember/index.js

```diff
@@ -1,141 +1,85 @@
-let isPercyRunning = true;
-let agentJS;
-// Capture fetch before it's mutated by Pretender
-let PercyFetch = window.fetch;
-
-async function fetchDOMLib() {
-  if (agentJS) return agentJS;
-
-  try {
-    return await PercyFetch('http://localhost:5338/percy-agent.js').then(res => res.text());
-  } catch (err) {
-    console.log(`[percy] Error fetching DOM library, disabling: ${err}`);
-    isPercyRunning = false;
+import utils from '@percy/sdk-utils';
+import { VERSION as emberVersion } from '@ember/version';
+import { VERSION as sdkVersion, PERCY_SERVER_ADDRESS } from '@percy/ember/meta';
+
+// Collect client and environment information
+const CLIENT_INFO = `@percy/ember/${sdkVersion}`;
+const ENV_INFO = [`ember/${emberVersion}`];
+
+if (window.QUnit) ENV_INFO.push(`qunit/${window.QUnit.version}`);
+if (window.mocha) ENV_INFO.push(`mocha/${window.mocha.version}`);
+
+// Maybe set the CLI API address from the environment
+utils.percy.address = PERCY_SERVER_ADDRESS;
+
+// Helper to generate a snapshot name from the test suite
+function generateName(assertOrTestOrName) {
+  if (assertOrTestOrName.test?.module?.name && assertOrTestOrName.test?.testName) {
```

Comment:
Are we going to run into any issues using these? Probably not -- just checking since ember apps are compiled by the consuming app

Reply:
Ember uses the version of `ember-cli-babel` defined in the app's/addon's package dependencies to compile. If we wanted to move that dep to be a devDep, we would then need to remove the Node 14 features.